src/Hl7.Fhir.Support.Poco.Tests/Rest/FhirClientMockTests.cs

```diff
@@ -19,14 +18,14 @@
 using System.Text;
 using System.Threading;
 using System.Threading.Tasks;
-using T = System.Threading.Tasks;
+using Task = System.Threading.Tasks.Task;
```

Comment:
Let's use T = System.Threading.Tasks with T.Task everywhere

(also this reference looks duplicate/obsolete now)

Reply:
I had to choose "Tasks.Task" and "using Tasks =", since you'll get compiler errors if a generic `T` is in scope otherwise.